src/script/util/DebugUtil.ts

```diff
@@ -110,7 +110,8 @@ export class DebugUtil {
   }
 
   async resetIdentity(): Promise<void> {
-    this.core['coreCryptoClient'].wipe();
+    const proteusService = this.core.service!.proteus;
+    await proteusService['cryptoClient'].debugResetIdentity();
   }
```

Comment:
should there be a proteus and coreCrypto version of this function?

Reply:
That's the beauty of it, the `CryptoClient` is a generic interface that wrap both CoreCrypto and Cryptobox, no need for the webapp to know what is underneath :)

Reply:
If coreCrypto is use (which is the case for MLS) MLS is handled as well in this method 👌